### PR TITLE
[PERFSCALE-2639] GCP: Move to common workflow

### PR DESCRIPTION
**Pull request by krishvoor**
This PR brings cluster install/uninstall under the same hood allowing users to reference this workflow.

### CODE REVIEW

**Comment by krishvoor** [comment]
/assign @paigerube14 @jtaleric 
cc: @svetsa-rh

**Comment by paigerube14** [comment]
Looks good to me, I'll let someone else review and comment to get this merged

File: ci-operator/step-registry/openshift-qe/installer/gcp/openshift-qe-installer-gcp-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+workflow:
+  as: openshift-qe-installer-gcp
+  steps:
+    post:
+    - chain: ipi-gcp-post
+    pre:
+    - chain: ipi-gcp-pre
+    - chain: create-infra-move-ingress-monitoring-registry
```
Comment (qiliRedHat) [inline ci-operator/step-registry/openshift-qe/installer/gcp/openshift-qe-installer-gcp-workflow.yaml:8]: I think the name 'openshift-qe-installer-gcp' make people who wants to reuse it thinks this is just for install. So 'create-infra-move-ingress-monitoring-registry' is not expected here. I suggest to put 'create-infra-move-ingress-monitoring-registry' in perfscale's test config.

Comment (liqcui) [reply ci-operator/step-registry/openshift-qe/installer/gcp/openshift-qe-installer-gcp-workflow.yaml:8]: +1

Comment (krishvoor) [reply ci-operator/step-registry/openshift-qe/installer/gcp/openshift-qe-installer-gcp-workflow.yaml:8]: Hi @qiliRedHat @liqcui  👋 ,
In a "self-managed" OCP, the infra components do not have designated nodes,
this chain simplifies that `create-infra-move-ingress-monitoring-registry`, in order to resonate a managed-service/ROSA like deployment, hence `create-infra-move-ingress-monitoring-registry` was captured into
the same workflow, which is originally inspired from https://github.com/openshift/release/pull/43601

**Comment by krishvoor** [comment]
/assign @bradmwilliams 

Hi @bradmwilliams ptal, this PR simplifies the usage of workflow for perf & scale team - thank you!

**Review by bradmwilliams — APPROVED** [review]

**Pull request merged**